eslint.config.mjs

```diff
@@ -116,6 +116,7 @@ export default tseslint.config([
       'vitest.workspace.mjs',
       // package specific ignores
       'packages/astro/src/astro-components/**/*.ts',
+      'packages/backend-api-client/**/*',
```

Comment:
Intentionally excluded from Prettier and ESLint as the code is generated and makes future reviews impossible.